lib/bn.js

```diff
@@ -1021,6 +1237,12 @@ BN.prototype.mul = function mul(num) {
   return this.mulTo(num, out);
 };
 
+// Multiply employing FFT
+BN.prototype.mulf = function mulf(num) {
+  var out = new BN(null);
+  return jumboMulTo(this, num, out);
```

Comment:
Ok. But it's useless:

That's way it's now:
```
Benchmarking: mul-jumbo
bn.js[FFT]#mul-jumbo x 3,957 ops/sec ±1.61% (8 runs sampled)
------------------------
Fastest is bn.js[FFT]#mul-jumbo
========================
Benchmarking: mul-jumbo
bn.js[FFT]#mul-jumbo x 3,963 ops/sec ±1.15% (7 runs sampled)
------------------------
Fastest is bn.js[FFT]#mul-jumbo
========================
Benchmarking: mul-jumbo
bn.js[FFT]#mul-jumbo x 3,934 ops/sec ±0.58% (8 runs sampled)
------------------------
Fastest is bn.js[FFT]#mul-jumbo
========================
Benchmarking: mul-jumbo
bn.js[FFT]#mul-jumbo x 3,965 ops/sec ±0.42% (7 runs sampled)
------------------------
Fastest is bn.js[FFT]#mul-jumbo
========================
Benchmarking: mul-jumbo
bn.js[FFT]#mul-jumbo x 3,965 ops/sec ±1.73% (8 runs sampled)
------------------------
Fastest is bn.js[FFT]#mul-jumbo
========================
Benchmarking: mul-jumbo
bn.js[FFT]#mul-jumbo x 3,965 ops/sec ±0.87% (9 runs sampled)
------------------------
Fastest is bn.js[FFT]#mul-jumbo
========================
Benchmarking: mul-jumbo
bn.js[FFT]#mul-jumbo x 3,777 ops/sec ±1.76% (9 runs sampled)
------------------------
Fastest is bn.js[FFT]#mul-jumbo
========================
Benchmarking: mul-jumbo
bn.js[FFT]#mul-jumbo x 3,827 ops/sec ±0.85% (9 runs sampled)
------------------------
Fastest is bn.js[FFT]#mul-jumbo
========================
Benchmarking: mul-jumbo
bn.js[FFT]#mul-jumbo x 3,824 ops/sec ±1.87% (8 runs sampled)
------------------------
Fastest is bn.js[FFT]#mul-jumbo
========================
Benchmarking: mul-jumbo
bn.js[FFT]#mul-jumbo x 4,016 ops/sec ±1.33% (8 runs sampled)
------------------------
Fastest is bn.js[FFT]#mul-jumbo
========================
```

That is with `out.words` being propagated:
```
Alexeys-MBP:benchmarks akudinkin$ for i in $(seq 1 10); do node index.js; done
Benchmarking: mul-jumbo
bn.js[FFT]#mul-jumbo x 3,523 ops/sec ±2.78% (9 runs sampled)
------------------------
Fastest is bn.js[FFT]#mul-jumbo
========================
Benchmarking: mul-jumbo
bn.js[FFT]#mul-jumbo x 3,448 ops/sec ±2.41% (9 runs sampled)
------------------------
Fastest is bn.js[FFT]#mul-jumbo
========================
Benchmarking: mul-jumbo
bn.js[FFT]#mul-jumbo x 3,511 ops/sec ±1.10% (9 runs sampled)
------------------------
Fastest is bn.js[FFT]#mul-jumbo
========================
Benchmarking: mul-jumbo
bn.js[FFT]#mul-jumbo x 3,455 ops/sec ±1.60% (9 runs sampled)
------------------------
Fastest is bn.js[FFT]#mul-jumbo
========================
Benchmarking: mul-jumbo
bn.js[FFT]#mul-jumbo x 3,461 ops/sec ±2.34% (8 runs sampled)
------------------------
Fastest is bn.js[FFT]#mul-jumbo
========================
Benchmarking: mul-jumbo
bn.js[FFT]#mul-jumbo x 3,552 ops/sec ±2.68% (9 runs sampled)
------------------------
Fastest is bn.js[FFT]#mul-jumbo
========================
Benchmarking: mul-jumbo
bn.js[FFT]#mul-jumbo x 3,372 ops/sec ±1.79% (8 runs sampled)
------------------------
Fastest is bn.js[FFT]#mul-jumbo
========================
Benchmarking: mul-jumbo
bn.js[FFT]#mul-jumbo x 3,424 ops/sec ±2.79% (9 runs sampled)
------------------------
Fastest is bn.js[FFT]#mul-jumbo
========================
Benchmarking: mul-jumbo
bn.js[FFT]#mul-jumbo x 3,451 ops/sec ±1.73% (9 runs sampled)
------------------------
Fastest is bn.js[FFT]#mul-jumbo
========================
Benchmarking: mul-jumbo
bn.js[FFT]#mul-jumbo x 3,327 ops/sec ±3.76% (9 runs sampled)
------------------------
Fastest is bn.js[FFT]#mul-jumbo
```